yamcs-web/src/main/webapp/src/app/shared/forms/ObjectSelector.ts

```diff
@@ -55,17 +55,32 @@ export class ObjectSelector implements ControlValueAccessor, OnChanges, OnDestro
     this.storageClient = yamcs.createStorageClient();
   }
 
+  // Called when bucket is changed
   ngOnChanges() {
-    if (this.instance && this.bucket) {
+    if (this.instance) {
       this.loadCurrentFolder();
     }
   }
 
+  // Called when breadcrumb is selected
   changePrefix(prefix: string) {
     this.loadCurrentFolder(prefix);
   }
 
   private loadCurrentFolder(prefix?: string) {
+    if (!this.bucket) {
+      // The object selector for remote files does not have a bucket.
```

Comment:
Think I would prefer that ObjectSelector remains purely used for selecting bucket objects.
Maybe make a RemoteFileSelector as part of the FileTransferModule?